### PR TITLE
Add nyc and coveralls

### PR DESCRIPTION
**Pull request by ralphtheninja**

### CODE REVIEW

**Comment by ralphtheninja** [comment]
Allow me to fix this to be at least 90%.

**Comment by ralphtheninja** [comment]
Actually, will try to get 100%.

File: test.js [hunk 1]
```diff
@@ -242,7 +242,78 @@ test('keys and values should not be serialized', function (t) {
...
+    }
+  }
+  var ld = new DeferredLevelDOWN(db)
+
+  ld.close(function (err) {
+    t.is(err.message, 'foo')
+  })
+})
+
+test('approximateSize is called on underlying store when store is open', function (t) {
```
Comment (vweevers) [inline test.js]: This is to test the "non-deferred" code path?

File: test.js [hunk 2]
```diff
@@ -284,3 +355,42 @@ test('iterators', function (t) {
...
+    var nextFirst = false
+
+    it.next(function (err, key, value) {
+      nextFirst = true
+      t.error(err, 'no error')
+      t.equal(key, 'key')
+      t.equal(value, 'value')
+    })
+
+    it.end(function (err) {
```
Comment (vweevers) [inline test.js]: Why does this test call `it.end` immediately after `it.next`? Does `deferred-leveldown` do something special in this case?

File: test.js (hunk 1 above)
Comment (ralphtheninja) [reply test.js]: Exactly

File: test.js (hunk 2 above)
Comment (ralphtheninja) [reply test.js]: Good question. I stole the code from the other test and tweaked it. It makes more sense to have them called immediately in the deferred case and instead we should probably do `it.end()` after `it.next()` has finished here. Or we can probably remove it completely even.

File: test.js (hunk 1 above)
Comment (ralphtheninja) [reply test.js]: Maybe I should rename the test to `non-deferred approximateSize`

**Pull request merged**